plaidml2/core/ffi.cc

```diff
@@ -51,6 +58,20 @@ void plaidml_init(plaidml_error* err) {
           el::Loggers::setVerboseLevel(level);
         }
       }
+      auto ctx = GlobalContext::getContext();
+      auto eventlog_str = vertexai::env::Get("PLAIDML_EVENTLOG_FILENAME");
+      if (ctx && eventlog_str.size()) {
```

Comment:
Why do we check for `ctx` to be non-null?

Reply:
That was a relic of the past code. I'll remove it.